pkg/xds/envoy/clusters/circuit_breaker_configurer.go

```diff
@@ -0,0 +1,94 @@
+package clusters
+
+import (
+	envoy_api "github.com/envoyproxy/go-control-plane/envoy/api/v2"
+	envoy_cluster "github.com/envoyproxy/go-control-plane/envoy/api/v2/cluster"
+	"github.com/golang/protobuf/ptypes/wrappers"
+
+	mesh_core "github.com/Kong/kuma/pkg/core/resources/apis/mesh"
+)
+
+func CircuitBreaker(circuitBreaker *mesh_core.CircuitBreakerResource) ClusterBuilderOpt {
```

Comment:
Can we change the name here to `OutlierDetectionConfigurer`? I think that names in `pkg/xds/envoy` should be Envoy names, not Kuma names. Look at RBAC. We call it `NetworkRBACConfigurer`, not `TrafficPermissionConfigurer`